typings/verbalexpression.d.ts

```diff
@@ -55,12 +55,12 @@ interface VerbalExpression extends RegExp {
     searchOneLine(enable?: boolean): VerbalExpression;
     repeatPrevious(): VerbalExpression;
     oneOrMore(): VerbalExpression;
-    multiple(value: string): VerbalExpression;
+    multiple(value: string, count?: number): VerbalExpression;
```

Comment:
I think this type definition is wrong for some reason?

Now we have `multiple(value, lower, upper)`, and this type only includes two arguments